### PR TITLE
Move domain names to debug level logging.

### PR DESCRIPTION
**Pull request by bemasc**
[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-networking/244)

### CODE REVIEW

File: src/socks-to-rtc/socks-to-rtc.ts [hunk 1]
```diff
@@ -437,8 +437,12 @@ module SocksToRtc {
       return this.tcpConnection_.receiveNext()
         .then(socks.interpretRequestBuffer)
         .then((request:socks.Request) => {
-          log.info('%1: received endpoint from SOCKS client: %2', [
-              this.longId(), JSON.stringify(request.endpoint)]);
+          // The domain name is very sensitive, so we keep it out of the
+          // info-level logs, which may be uploaded.
+          log.debug('%1: received address from SOCKS client: %2', [
+              this.longId(), request.endpoint.address]);
+          log.info('%1: received port from SOCKS client: %2', [
```
Comment (iislucas) [inline src/socks-to-rtc/socks-to-rtc.ts]: This is probably going to be a lot of 80 and 443 right? What do you think about just dropping this info message, and print the full address at debug level.  Likewise with rtc to net?

**Comment by trevj** [comment]
<sup>**[src/socks-to-rtc/socks-to-rtc.ts, line 444 [r1]](https://reviewable.io:443/reviews/uproxy/uproxy-networking/244#-Jn8AJfDXgsrtqHsngsv-r1-444)** ([raw file](https://github.com/uproxy/uproxy-networking/blob/28657fd792880cbf590ebc16a873a1fc4e172712/src/socks-to-rtc/socks-to-rtc.ts#L444)):</sup>
Agreed. Let's just print the whole thing, but at DEBUG.

Can you also add a new INFO line at the top of `linkSocketAndChannel_`? Then we can still identify when the SOCKS handshake has failed (or hung) for some reason.

---

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-networking/244)

**Comment by bemasc** [comment]
<sup>**[src/socks-to-rtc/socks-to-rtc.ts, line 444 [r1]](https://reviewable.io:443/reviews/uproxy/uproxy-networking/244#-Jn8AJfDXgsrtqHsngsv-r1-444)** ([raw file](https://github.com/uproxy/uproxy-networking/blob/28657fd792880cbf590ebc16a873a1fc4e172712/src/socks-to-rtc/socks-to-rtc.ts#L444)):</sup>
Done and Done.

---

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-networking/244)

**Comment by trevj** [comment]
:+1: and thanks for looking at this!

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy-networking/244)

**Pull request merged**